server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -202,47 +202,50 @@ def validate_cfme_parameters(deployment)
           deployment.errors[:cfme_root_password] << _('CloudForms deployments must specify a root password for the CloudForms machines')
         end
 
-        if deployment.rhev_export_domain_name.empty?
-          deployment.errors[:rhev_export_domain_name] << _('CloudForms deployments must specify a RHV export domain name')
-        elsif (deployment.deploy_rhev && deployment.rhev_export_domain_name == deployment.rhev_storage_name) ||
-          (deployment.rhev_is_self_hosted && deployment.rhev_export_domain_name == deployment.hosted_storage_name)
-          deployment.errors[:rhev_export_domain_name] << _('RHV export data domain name is not unique')
-        end
+        if deployment.cfme_install_loc == 'RHEV'
```

Comment:
Terrible diff, but this (and it's matching "end") is the only non-test code change.